### ISSUE TITLE
Dev: Web - Console error Provide a valid string for accessibilityLabel prop

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#26003]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Go to any money request report.
2. Click on the amount such that the edit amount screen opens up.

## Expected Result:
 No console error

## Actual Result:
Getting the console error:
Warning: Failed prop type: Provide a valid string for accessibilityLabel prop when accessible is true

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** Dev: 
**Reproducible in staging?:** n/a
**Reproducible in production?:** n/a
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

![Screenshot 2023-08-18 at 12 17 01 AM](https://github.com/Expensify/App/assets/93399543/0d5a4a8b-449c-4c74-b66a-051947b51d22)

**Expensify/Expensify Issue URL:**
**Issue reported by:** @allroundexperts 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1692300040853639

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by AmjedNazzal** [comment]
I'm unable to repo this.

**Comment by JmillsExpensify** [comment]
Alright, closing.
